histolab/tile.py

```diff
@@ -166,6 +166,53 @@ def save(self, path: Union[str, bytes, os.PathLike]) -> None:
         Path(path).parent.mkdir(parents=True, exist_ok=True)
         self._image.save(path)
 
+    @lazyproperty
+    def tissue_mask(self) -> np.ndarray:
+        """Return a binary mask representing the tissue in the tile.
+
+        Returns
+        -------
+        np.ndarray
+            Binary mask representing the tissue in the tile.
+
+        Notes
+        -----
+        In order to address an issue with the calculation of the tissue mask in tiles
+        where the tissue covers all the tile area, the filters are applied on the tile
+        with a white border of 10px, and then the border is removed from the returned
+        mask.
+        """
+        np_tile = np.array(self._image)
+
+        border_thickness = 10
+
+        # create a "white" array 20px bigger than the tile
```

Comment:
in the docstring there's a 10px reference but here I see 20px. I'm confuse 😄

Reply:
it's a 10px border on each X and Y side, so in total the container is 20px bigger on the X and Y axis